src/manifest.json

```diff
@@ -3,9 +3,14 @@
       "cockpit": "215"
   },
 
-  "tools": {
+  "menu": {
```

Comment:
That of course opens up a nice round of bikeshedding. I thought "menu" was like "functional subsystems", like containers, machines, users; and "tools" more like "programs". Conceptually, a file browser feel similar to a terminal to me.

Reply:
I am also not set on it, I think @garrett and @allisonkarlitskaya felt it should be under system let's discuss it here